config/initializers/15_cloud_config.rb

```diff
@@ -0,0 +1,15 @@
+module CloudConfig
+  if ENV.has_key?('OSS_USE_AZURE') && ENV['OSS_USE_AZURE'].to_i == 1
```

Comment:
__[rubocop]__
* Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.